third_party/winuwp_h264/H264Encoder/H264Encoder.cc

```diff
@@ -312,6 +318,10 @@ ComPtr<IMFSample> WinUWPH264EncoderImpl::FromVideoFrame(const VideoFrame& frame)
   return sample;
 }
 
+LONGLONG WinUWPH264EncoderImpl::GetFrameTimestampHns(const VideoFrame& frame) const {
+  return ((frame.timestamp() - startTime_) / 90) * 1000 * 10;
```

Comment:
Can you please add a comment and reference https://tools.ietf.org/html/rfc6184#page-11 which states that:
> A 90 kHz clock rate MUST be used.

This is the actual reason why we have this magical "90" number here. The actual formula is:
```
timestamp_100ns = timestamp_90kHz / {90'000 kHz} * {10'000'000 hns/sec}
```